File: src/main/java/jp/artan/japanesefoodmod/raw_materials/wood/RawMaterialsFruitLeaves.java
package jp.artan.japanesefoodmod.raw_materials.wood;

import jp.artan.japanesefoodmod.common.wood.FruitLeaves;
import jp.artan.japanesefoodmod.init.CreativeTabsInit;
import jp.artan.japanesefoodmod.raw_materials.JapaneseFoodRawMaterials;

public class RawMaterialsFruitLeaves extends FruitLeaves {

    public RawMaterialsFruitLeaves(String name) {
        super(name);
        this.setCreativeTab(CreativeTabsInit.fruitTreeCreativeTab);
    }
}

File: src/main/java/jp/artan/japanesefoodmod/raw_materials/wood/RawMaterialsFruitLeaves.kt
package jp.artan.japanesefoodmod.raw_materials.wood

import jp.artan.japanesefoodmod.common.wood.FruitLeaves
import jp.artan.japanesefoodmod.init.CreativeTabsInit

class RawMaterialsFruitLeaves(name: String) : FruitLeaves(name) {
    init {
        this.setCreativeTab(CreativeTabsInit.fruitTreeCreativeTab)
    }
}
